packages/wrangler/src/versions/upload.ts

```diff
@@ -567,18 +566,7 @@ async function ensureQueuesExist(config: Config) {
 
 	const queueNames = producers.concat(consumers);
 	for (const queue of queueNames) {
-		try {
-			await getQueue(config, queue);
-		} catch (err) {
-			const queueErr = err as FetchError;
-			if (queueErr.code === 11000) {
-				// queue_not_found
-				throw new UserError(
-					`Queue "${queue}" does not exist. To create it, run: wrangler queues create ${queue}`
-				);
-			}
-			throw err;
-		}
+		await getQueue(config, queue);
```

Comment:
could this loop be replaced with a single call to get multiple queues by name?

Reply:
Yap, just requires a few utility functions. Will do that.